bucket.go

```diff
@@ -160,12 +170,20 @@ func OpenBucket(urlStr string, mode OpenMode) (bucket *Bucket, err error) {
 	db.SetMaxOpenConns(ifelse(inMemory, 1, kMaxOpenConnections))
 
 	bucket = &Bucket{
-		url:         urlStr,
-		sqliteDB:    db,
-		collections: make(map[sgbucket.DataStoreNameImpl]*Collection),
-		inMemory:    inMemory,
-		serial:      serial,
+		url:             urlStr,
+		sqliteDB:        db,
+		collections:     make(map[sgbucket.DataStoreNameImpl]*Collection),
+		collectionFeeds: make(map[sgbucket.DataStoreNameImpl][]*dcpFeed),
+		mutex:           &sync.Mutex{},
+		nextExp:         new(uint32),
```

Comment:
I'm not familiar with this usage - what's the advantage of setting new(uint32) instead of leaving nil?

Reply:
This is just initializing a an int pointer to 0, is there a more canonical way without writing a function?